### PR TITLE
Release 3.0.4

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.0.4 (2019-06-07)
+
+- Fixed the modal component to have correct accessibility tab controls. [#46](https://github.com/blackbaud/skyux-modals/pull/46)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md]: Fixed the modal component to correct accessibility tab controls.

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Also, might want to add a shout out to Jack.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 3.0.4 (2019-06-07)
+
+- FFixed the modal component to correct accessibility tab controls. [#46](https://github.com/blackbaud/skyux-modals/pull/46) (Thanks @Blackbaud-JackMcElhinney)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md]: Double FF?!

Comment (Blackbaud-TrevorBurch) [reply CHANGELOG.md]: Doh. Fixed

**Pull request merged**